tests/integration_tests/migrations/f84fde59123a_update_charts_with_old_time_comparison__test.py

```diff
@@ -161,6 +161,70 @@
         }
     ],
 }
+params_v1_other_than_custom_false: dict[str, Any] = {
```

Comment:
Would you mind extracting the common parts between `params_v1_with_custom`, `params_v1_with_custom`, `params_v1_other_than_custom_false`, `params_v2_with_custom`, `params_v2_other_than_custom`, `params_v2_other_than_custom_false` and just override the necessary properties for each variable to avoid code duplication and increase readability?